tests/integration/tests/test-home-page.js

```diff
@@ -0,0 +1,80 @@
+/* global browser */
+/* global $ */
+/* global $$ */
+
+"use strict";
+
+const assert = require("assert");
+const HomePage = require("../pages/desktop/home.page");
+const NavBar = require("../regions/navbar.region");
+const UserDashboardPage = require("../pages/desktop/dashboard.page");
+
+describe("Firefox Monitor homepage", function() {
+  //this.retries(2);
+
+  beforeEach(function() {
+    browser.url("http://localhost:6060");
+  });
+
+  it("should load the latest breach card", function() {
+    const homePage = HomePage;
+
+    homePage.waitForPageToLoad();
+    assert(
+        homePage.breachCard.latestBreachCard.isDisplayed(),
+        "Latest Breach card not found"
+    );
+  });
+
+  it("should look like normal", function() {
+    expect(browser.checkFullPageScreen("Home_Page", {})).to.be.within(0, 5);
+  });
+
+  it("should load correct number of breaches from an email input", function() {
+    const homePage = HomePage;
+
+    homePage.waitForPageToLoad();
+    homePage.breachEmailAddress.setValue(global.primaryEmail);
+    const scanResults = homePage.checkBreachesButton;
+    expect(scanResults.breachCards.length)
+      .to
+      .equal(Number(scanResults.numberOfBreaches));
+  });
+
+  it("should allow secondary email to be added", function() {
+    const homePage = HomePage;
+    const navBar = NavBar;
+
+    homePage.waitForPageToLoad();
+    navBar.signIn.click();
+    // FxA login
+    // Check if fxa email is needed, if error, just continue
+    try {
+      $(".email").waitForExist(5000);
+      $(".email").click();
+      $(".email").setValue(global.primaryEmail);
+      $("#submit-btn").click();
+    } catch (error) {
```

Comment:
Curious, why would this error?

Reply:
Sometimes the email doesn't show as monitor passes the email you used for checking breaches into the fxa sign in